### PR TITLE
Added fragment block and home page styling

### LINKED ISSUE #8: Fragment: Important Safety Information
I think it could be done as autoblocking fragment

### LINKED ISSUE #20: Home content styling
Screenshot 2023-10-09 at 17 55 31

### PR DESCRIPTION
**Pull request by lansami**
Fix #8 

Test URLs:
- Before: https://main--bevespi--hlxsites.hlx.page/
- After: https://issue-20-home-styling--bevespi--hlxsites.hlx.page/

### CODE REVIEW

**Review by paolomoz — CHANGES_REQUESTED** [review]
@lansami it seems that styling is not done yet. I would suggest to complete before review
see screenshot in comments

**Comment by paolomoz** [comment]
Screenshot 2023-10-13 at 17 43 14

**Comment by paolomoz** [comment]
@lansami I think it needs some styling adjustments, see screenshot:

Screenshot 2023-10-16 at 10 56 13

**Comment by paolomoz** [comment]
There are some style fix required, see screenshot below:

- Resources and support has a different font face
- pink icon is missing (I see you noted it is known, but I would prefer to include pictures in this PR; any reason to not include them?)
- Explore the services.. text; should not be upper case, and should be 16px not 17px
- Based on what we decided for the project, font size is in rem with font root size 10px and everything else (padding, margin, etc..) is px
- Padding top bottom. should be increased a bit

Screenshot 2023-10-16 at 15 54 57

**Comment by paolomoz** [comment]
Style fixes for the fragment part:

- Orange title font size should be 18px not 22px, 14px for mobile
- text font size should be 15px not 16px, 13px for mobile
- collapsed fragment font size should be 15px not 9px
- fragment should expand as soon as its title get above the fold, not later
- scroll when clicking on the + should be slower
- we changed max-width to 1200px for header, footer and content (fragment still at 1002px)
- after scrolling on mobile the title should be close to the to of the window (there is some padding now)
- ul should not have padding left
- bold text is not rendered

Screenshot 2023-10-16 at 16 00 45

**Review by paolomoz — APPROVED** [review]
@lansami approved with comments

File: blocks/fragment/fragment.css [hunk 1]
```diff
@@ -1 +1,160 @@
...
+}
+
+.sticky-fragment-block .sticky-fragment-header .sticky-fragment-scroll-button::after {
+    content: "";
+    width: 2.4px;
+    height: 16px;
+    background-color: var(--sticky-fragment-header-color);
+    position: absolute;
+    top: 26%;
+    left: 46%;
```
[the remark this answers is not in the document]
Comment (lansami) [reply blocks/fragment/fragment.css:154]: I've updated the styling to be more specific to the "Important Safety Information" fragment

File: blocks/fragment/fragment.css [hunk 2]
```diff
@@ -1 +1,160 @@
...
+}
+
+.fragment-container > div a {
+    color: var(--information-link-color) !important;
+    text-decoration: underline !important;
+}
+
+.fragment-container div p em + picture img {
+    max-width: 10px;
+}
```
[the remark this answers is not in the document]
Comment (lansami) [reply blocks/fragment/fragment.css]: updated to be more specific to the "Important Safety Information"

**Pull request merged**